grass7/imagery/i.landsat/i.landsat.qa/i.landsat.qa.html

```diff
@@ -8,7 +8,8 @@ <h2>DESCRIPTION</h2>
 can be used in <a href="https://grass.osgeo.org/grass-stable/manuals/r.reclass.html">r.reclass</a> for filtering the 
 QA band according to pixel quality characteristics the user defines as 
 unacceptable. It works with both Collection 1 and Collection 2 data from 
-Landsat 8 OLI/TIRS, 4-7 TM/ETM+, and 1-5 MSS sensors. 
+Landsat 8 OLI/TIRS, 4-7 TM/ETM+. The <b>dataset</b> the QA band belongs to 
+is specified like in <a href="ihttps://grass.osgeo.org/grass-stable/manuals/addons/i.landsat.download.html">i.landsat.download</a>. 
```

Comment:
```suggestion
is specified like in <a href="i.landsat.download.html">i.landsat.download</a>. 
```
(only GRASS-core manual pages get the full URL; addons the relative one)